plugins/module_utils/network/vyos/argspec/ntp_global/ntp_global.py

```diff
@@ -49,6 +49,7 @@ class Ntp_globalArgs(object):  # pylint: disable=R0903
                                 "dynamic",
                                 "preempt",
                                 "prefer",
+                                "pool",
```

Comment:
nit: this list of options was alphabetized prior to this change. This applies elsewhere in this change as well.